app/assets/src/components/BulkUploadImport.jsx

```diff
@@ -335,15 +346,19 @@ class BulkUploadImport extends React.Component {
     } else {
       errors.project = "Please select a project";
     }
-    if (this.refs.bulk_path) {
-      if (this.refs.bulk_path.value === "") {
+
+    if (!this.state.localUploadMode) {
+      if (this.refs.bulk_path) {
+        if (this.refs.bulk_path.value === "") {
```

Comment:
Do you need this many errors? You can just say that it is invalid and do
```
if (!this.refs.bulk_path
	 || this.refs.bulk_path.value === ""
	 || !this.filePathValid(this.refs.bulk_path.value))
```
Probably even put the second condition inside as part of the validation function.

Reply:
👍